river/tree/__init__.py

```diff
@@ -39,6 +39,10 @@
 * Define properties to access leaf prediction strategies, split criteria, and other
 relevant characteristics.
 
+**2. Stochastic Gradient Trees**
+
+TODO
```

Comment:
I need to do that :P

@henrygouk, please feel free to give us any suggestions to introduce the SGTs :D